tests/calculateRank.test.js

```diff
@@ -2,17 +2,45 @@ require("@testing-library/jest-dom");
 const calculateRank = require("../src/calculateRank");
 
 describe("Test calculateRank", () => {
-  it("should calculate rank correctly", () => {
+  it("new user gets B rank", () => {
     expect(
       calculateRank({
-        totalCommits: 100,
-        totalRepos: 5,
-        followers: 100,
-        contributions: 61,
-        stargazers: 400,
-        prs: 300,
-        issues: 200,
+        totalRepos: 0,
+        totalCommits: 0,
+        contributions: 0,
+        followers: 0,
+        prs: 0,
+        issues: 0,
+        stargazers: 0,
       }),
-    ).toStrictEqual({ level: "A+", score: 49.16605417270399 });
+    ).toStrictEqual({level: "B", score: 100.});
+  });
+
+  it("average user gets A rank", () => {
+    expect(
+      calculateRank({
+        totalRepos: 10,
+        totalCommits: 500,
+        contributions: 500,
+        followers: 50,
+        prs: 12,
+        issues: 13,
+        stargazers: 100,
+      }),
+    ).toStrictEqual({"A", score: 50.});
```

Comment:
```suggestion
    ).toStrictEqual({level: "A", score: 50.});
```

Otherwise, the tests will throw a `Unexpected token (30:24)` error.